packages/asc-ui/src/components/TextField/TextField.tsx

```diff
@@ -24,15 +26,19 @@ const TextField = ({
   value,
   keepFocus,
   blurOnEscape,
-  focusOnRender,
   errorMessage,
   labelStyle,
   errorStyle,
   id: idProp,
+  autoFocus,
   ...otherProps
 }: TextFieldProps) => {
   const uid = useUID()
   const id = idProp || uid
+  const inputRef = useRef<HTMLInputElement>(null)
```

Comment:
Why do you need `(null)` here?

Reply:
Otherwise we end up with a mutable ref. In this case the ref is defined, just possibly containing a null value. In this case I pulled it from the [React docs](https://reactjs.org/docs/hooks-reference.html#useref).